core/src/main/java/greencity/mapping/MultipartBase64ImageMapper.java

```diff
@@ -40,7 +37,14 @@ protected MultipartFile convert(String image) {
             ImageIO.write(bufferedImage, "png", tempFile);
             FileItem fileItem = new DiskFileItem("mainFile", Files.probeContentType(tempFile.toPath()),
                 false, tempFile.getName(), (int) tempFile.length(), tempFile.getParentFile());
-            fileItem.getOutputStream();
+            InputStream input = new FileInputStream(tempFile);
```

Comment:
- This is not really optimal to copy byte by byte.
- IOStreams should be closed (using try-with-resources block)

To sum up: you can use https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/IOUtils.html#copy-java.io.InputStream-java.io.OutputStream-

Here and in two more places